### PR TITLE
Change netkvm param name for jumbo in windows

### PR DESCRIPTION
**Pull request by kimi1978wy**
For product change, need change MTU to *JumboPacket

id:1898870

Signed-off-by: Yu Wang <wyu@redhat.com>

### CODE REVIEW

**Comment by kimi1978wy** [comment]
(1/2) type_specific.io-github-autotest-qemu.enable_scatter_windows: PASS (531.30 s)
 (2/2) type_specific.io-github-autotest-qemu.netkvm_change_param_value_test.modify_directly: PASS (370.45 s)

**Comment by kimi1978wy** [comment]
@yanglei-rh @PaulYuuu  , pls help to review, thanks a lot

File: qemu/tests/cfg/enable_scatter_windows.cfg [hunk 1]
```diff
@@ -19,7 +19,7 @@
     autoit_name = "AutoIt3_%PROCESSOR_ARCHITECTURE%.exe"
     install_winpcap_cmd = "WIN_UTILS:\${autoit_name} WIN_UTILS:\install_winpcap.au3"
 
-    param_names = "Offload.TxLSO Offload.TxLSO MTU"
+    param_names = "Offload.TxLSO Offload.TxLSO *JumboPacket"
```
Comment (PaulYuuu) [inline qemu/tests/cfg/enable_scatter_windows.cfg:22]: In order to maintain downward compatibility, please introduce a judgment to determine whether to use MTU or JumboPacket.

**Comment by kimi1978wy** [comment]
@PaulYuuu @yanglei-rh updated, pls review it again, thanks

Tested on win10, and win2019  with build189/190/191, all passed 

(1/1) type_specific.io-github-autotest-qemu.netkvm_change_param_value_test.modify_directly: PASS (377.48 s)
 (1/1) type_specific.io-github-autotest-qemu.enable_scatter_windows: PASS (457.07 s)

**Comment by yanglei-rh** [comment]
LGTM

virtio-win-prewhql-0.1-191.iso:
(2/3) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (531.28 s)
(3/3) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.netkvm_change_param_value_test.modify_directly.q35: PASS (396.27 s)
virtio-win-prewhql-0.1-189.iso:
(1/2) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.netkvm_change_param_value_test.modify_directly.q35: PASS (403.28 s)
(2/2) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (541.29 s)

File: qemu/tests/netkvm_change_param_value_test.py [hunk 2]
```diff
@@ -64,11 +75,16 @@ def start_test(param_name, param_value):
...
 
     virtio_win.prepare_netkvmco(vm)
     for name in param_names:
         attr_name = "param_values_%s" % name
         param_values = params.get(attr_name, param_values_default)
         for value in param_values.split():
+            if name == "*JumboPacket" and driver_version <= 189:
+                continue
+            elif name == "MTU" and driver_version > 189:
+                continue
```
Comment (PaulYuuu) [inline qemu/tests/netkvm_change_param_value_test.py:89]:
```suggestion
        param_values = params.get(attr_name, param_values_default)
        if driver_version <= 189:
            param_values.remove("*JumboPacket")
        else:
            param_values.remove("MTU")
        for value in param_values.split():
```
Would it be easier?

**Comment by kimi1978wy** [comment]
(1/1) type_specific.io-github-autotest-qemu.netkvm_change_param_value_test.modify_directly: PASS (363.28 s)
 (1/1) type_specific.io-github-autotest-qemu.netkvm_change_param_value_test.modify_directly: PASS (367.63 s)

**Comment by kimi1978wy** [comment]
@PaulYuuu Updated, pls help to review again. Thanks

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**